sequence_tools.py

```diff
@@ -0,0 +1,145 @@
+# Importing modules
+from typing import Union
+import source.fastq_read as fr
+import source.protein_tools as pt
+import source.nucleic_acids_tools as nat
+
+
+# Function for filtering FASTQ file
+def fastq_filter(seqs: dict, gc_bound: Union[tuple, int, float] = (0, 100),
+                 length_bound: Union[tuple, int, float] = (0, 2**32),
+                 quality_threshold: Union[int, float] = 0) -> dict:
+    """
+    This function work with FASTQ files and filters them by
+    GC content, length and Q-score.
+
+    Arguments:
+    - seqs (dict): dictionary consisting of fastq sequences. The structure is as follows.
+    Key: string, sequence name. Value: tuple of two strings (sequence and quality).
+    - gc_bound (tuple, int, float): tuple of required range of GC percentage (inclusive),
+    num or float if only higher border of the range is needed (exclusive).
+    - length_bound (tuple, int, float): tuple of required range of sequences length (inclusive),
+    num or float if only higher border of the range is needed (exclusive).
+    - quality_threshold (int): int of lowest level of Q-score (inclusive).
+
+    Output:
+    - dictionary of those samples, which match all conditions.
+    """
+    if not len(seqs) > 0:
+        raise ValueError('There are no fastq sequences')
+    seqs_type = isinstance(seqs, dict)
+    gc_bound_type = isinstance(gc_bound, (tuple, int, float))
+    length_bound_type = isinstance(length_bound, (tuple, int, float))
+    quality_thr_type = isinstance(quality_threshold, (int, float))
+    if not (seqs_type and gc_bound_type and length_bound_type and quality_thr_type):
+        raise ValueError('Your arguments are not suitable!')
+    result_dict = seqs.copy()
+    for value in seqs:
+        fastq = seqs[value][0]
```

Comment:
Если учесть комментарий выше, то 
```suggestion
        fastq = value[0]
```